### PR TITLE
Upgrade to spring-cloud-dependencies:2021.0.0

### PR DESCRIPTION
**Pull request by moarychan**
As title.
Related issue:https://github.com/Azure/azure-sdk-for-java/issues/25581, it is compatible with Spring Boot: 2.6.1
Ref: https://spring.io/blog/2021/12/02/spring-cloud-2021-0-0-codename-jubilee-has-been-released

### CODE REVIEW

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md:15]: aad-b2c does not has relationship with spring-cloud-dependencies.

File: sdk/spring/azure-identity-spring/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-identity-spring/CHANGELOG.md:15]: `spring-cloud-dependencies:2.5.4` -> `spring-cloud-dependencies:2021.0.0`

File: sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md [hunk 3]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-active-directory/CHANGELOG.md:15]: Same here.

File: sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md (hunk 1 above)
Comment (chenrujun) [reply sdk/spring/azure-spring-boot-starter-active-directory-b2c/CHANGELOG.md:15]: `spring-cloud-dependencies:2.5.4` -> `spring-cloud-dependencies:2021.0.0`

File: sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md [hunk 4]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md:15]: Same here

File: sdk/spring/azure-identity-spring/CHANGELOG.md (hunk 2 above)
Comment (chenrujun) [reply sdk/spring/azure-identity-spring/CHANGELOG.md:15]: Could you please add `This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1`

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md [hunk 5]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:15]: Same here.

File: sdk/spring/azure-spring-boot-starter-keyvault-secrets/CHANGELOG.md [hunk 6]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-secrets/CHANGELOG.md:15]: Mark.

File: sdk/spring/azure-spring-boot-starter-servicebus-jms/CHANGELOG.md [hunk 7]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-servicebus-jms/CHANGELOG.md:15]: Mark

File: sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md [hunk 8]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-boot-starter/CHANGELOG.md [hunk 9]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-boot/CHANGELOG.md [hunk 10]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md [hunk 11]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-context/CHANGELOG.md [hunk 12]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-context/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-messaging/CHANGELOG.md [hunk 13]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-messaging/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-starter-cache/CHANGELOG.md [hunk 14]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-cache/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/CHANGELOG.md [hunk 15]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-starter-eventhubs/CHANGELOG.md [hunk 16]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-eventhubs/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-starter-servicebus/CHANGELOG.md [hunk 17]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-servicebus/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-starter-storage-queue/CHANGELOG.md [hunk 18]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-storage-queue/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-storage/CHANGELOG.md [hunk 19]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-storage/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-stream-binder-eventhubs/CHANGELOG.md [hunk 20]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-eventhubs/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-core/CHANGELOG.md [hunk 21]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-core/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md [hunk 22]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-topic/CHANGELOG.md [hunk 23]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-topic/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md [hunk 24]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-integration-core/CHANGELOG.md [hunk 25]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-core/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-integration-eventhubs/CHANGELOG.md [hunk 26]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-eventhubs/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-integration-servicebus/CHANGELOG.md [hunk 27]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-servicebus/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-integration-storage-queue/CHANGELOG.md [hunk 28]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-storage-queue/CHANGELOG.md:15]: M

File: sdk/spring/azure-spring-integration-test/CHANGELOG.md [hunk 29]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
+- Upgrade to [spring-boot-dependencies:2.6.1](https://repo1.maven.org/maven2/org/springframework/boot/spring-boot-dependencies/2.6.1/spring-boot-dependencies-2.6.1.pom).
+- Upgrade to [spring-cloud-dependencies:2.5.4](https://repo1.maven.org/maven2/org/springframework/cloud/spring-cloud-dependencies/2021.0.0/spring-cloud-dependencies-2021.0.0.pom).
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-test/CHANGELOG.md:15]: M

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md [hunk 30]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-autoconfigure/CHANGELOG.md:4]: How about spring-cloud's version?

File: sdk/spring/azure-spring-cloud-context/CHANGELOG.md [hunk 31]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-context/CHANGELOG.md:4]: Same here.

File: sdk/spring/azure-spring-cloud-messaging/CHANGELOG.md [hunk 32]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-messaging/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-starter-cache/CHANGELOG.md [hunk 33]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-cache/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/CHANGELOG.md [hunk 34]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-eventhubs-kafka/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-starter-eventhubs/CHANGELOG.md [hunk 35]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-eventhubs/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-starter-servicebus/CHANGELOG.md [hunk 36]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-servicebus/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-starter-storage-queue/CHANGELOG.md [hunk 37]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-starter-storage-queue/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-storage/CHANGELOG.md [hunk 38]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-storage/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-stream-binder-eventhubs/CHANGELOG.md [hunk 39]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-eventhubs/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-core/CHANGELOG.md [hunk 40]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-core/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md [hunk 41]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-stream-binder-servicebus-topic/CHANGELOG.md [hunk 42]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-servicebus-topic/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md [hunk 43]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-integration-core/CHANGELOG.md [hunk 44]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-core/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-integration-eventhubs/CHANGELOG.md [hunk 45]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-eventhubs/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-integration-servicebus/CHANGELOG.md [hunk 46]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-servicebus/CHANGELOG.md:4]: m

File: sdk/spring/azure-spring-integration-storage-queue/CHANGELOG.md [hunk 47]
```diff
@@ -10,6 +10,10 @@
 
 ### Other Changes
 
+### Dependency Upgrades
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-storage-queue/CHANGELOG.md:13]: his release is compatible with ...

File: sdk/spring/azure-spring-integration-test/CHANGELOG.md [hunk 48]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-integration-test/CHANGELOG.md:4]: m

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md [hunk 49]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 2.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.0-2.5.4, 2.6.0-2.6.1, and Spring Cloud 2021.0.0.
```
Comment (chenrujun) [inline sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md:4]: Good. This version does not support Spring Cloud 2020.0.

Comment (chenrujun) [reply sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md:4]: How about the modules that depend on this one?
image

**Review by chenrujun — APPROVED** [review]
LGTM.

File: sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md (hunk 49 above)
Comment (moarychan) [reply sdk/spring/azure-spring-cloud-stream-binder-test/CHANGELOG.md:4]: Tested the three binders，using spring boot 2.5.7 spring cloud 2020.0.5 and spring boot 2.6.1 spring cloud 2021.0.0, they all work normally.

File: sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md [hunk 50]
```diff
@@ -1,6 +1,7 @@
 # Release History
 
 ## 3.12.0-beta.1 (Unreleased)
+This release is compatible with Spring Boot 2.5.5-2.5.8, 2.6.0-2.6.1.
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md:4]: Why the libraries are not working with the lower version of Spring Boot 2.5.x?

**Review by saragluna — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md (hunk 50 above)
Comment (moarychan) [reply sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md:4]: Due to netty break change `java.lang.NoSuchMethodError: io.netty.internal.tcnative.SSLContext.setCurvesList(J[Ljava/lang/String;)Z`, eventhubs and cosmos sample will not work in Spring Boot 2.5.4 version.

Comment (saragluna) [reply sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md:4]: Could you please offer more details here? Like EH or SB is using which netty version and such netty version is brought by which azure-core lib? And this SSLContext.setCurvesList(J[Ljava/lang/String;)Z is added in which netty version?

Comment (moarychan) [reply sdk/spring/azure-spring-boot-starter-cosmos/CHANGELOG.md:4]: LIst the following dependencies according to the dependency of the error message:
- azure-messaging-eventhubs:5.10.3 -> azure-core:1.22.0->netty-tcnative-boringssl-static:2.0.43.Final
- spring booot 2.5.5 -> netty-tcnative:2.0.43.Final
- spring booot 2.5.4 -> netty-tcnative:2.0.40.Final, currently found exception in Cosmos, Eventhusb samples

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**